src/FormField/Lookup.php

```diff
@@ -143,7 +143,7 @@ public function init()
 
         $vp->set(function ($p) {
             $f = $p->add('Form');
-            $f->setModel($this->model);
+            $f->setModel($this->model->newInstance());
```

Comment:
this is the other part of code that i mention in the first comment of PR.

When we use an extended model, like : 

- Model Customer extends Model ( master model )
- Model CustomerActive extends Customer ( is an empty model with only the condition state = active )

if i put in standard LookUp it works perfect, but if i add button plus functionality it will load a new model with the defined model condition applied. At this point it will open a form with the first record matching the defined condition and not a new record.

If we click the button we want to add a new Model of that kind and i think ->newInstance is correct